src/PolykeyClient.ts

```diff
@@ -88,6 +119,7 @@ class PolykeyClient {
   public readonly session: Session;
   public readonly rpcClientClient: RPCClient<typeof clientClientManifest>;
 
+  protected ws: WebSocketClient;
```

Comment:
Let's keep names consistent with the class. 
```suggestion
  protected webSocketClient: WebSocketClient;
```

Reply:
Done.